### PR TITLE
Test another branch

### PR DESCRIPTION
**Pull request by aaronsn**
Trying out something that might cause a merge conflict

### CODE REVIEW

**Comment by clhenrick** [comment]
I am commenting on this PR like a robot.

**Comment by ldbnr4** [comment]
Tissues for your merge conflict issues. LGTM

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Hello World!!! - Krista
\ No newline at end of file
+# Hello World!!! - Krista
+# Curious how merge conflicts work
```
Comment (ldbnr4) [inline README.md]: Github has a nice online editor to resolve simple merge conflicts

**Review by ldbnr4 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aaronsn) [reply README.md]: I saw that, cool!

**Pull request merged**